tools/nnpackage_tool/tflite2circle/tflite2circle.sh

```diff
@@ -73,7 +73,7 @@ name=${tflite_base%.*}
 # convert
 
 mkdir -p "${outdir}"
-${flatc} -o ${outdir} --defaults-json --strict-json -t ${tflite_schema} -- $1
+${flatc} -o ${outdir} --strict-json -t ${tflite_schema} -- $1
```

Comment:
Is there any reason to remove this option? All default values are unnecessary?

Reply:
@mhs4670go Yes, default value in json is for human. flatbuffer does not save value for default values.